tutorials/dagster-serverless-jobs/index.mdx

```diff
@@ -0,0 +1,190 @@
+        API Docs: https://github.com/HackerNews/API#new-top-and-best-stories
+        """
+        top_story_ids = requests.get(
+            "https://hacker-news.firebaseio.com/v0/topstories.json"
+        ).json()
+        return top_story_ids[:10]
+
+
+    # asset dependencies can be inferred from parameter names
+    @asset
+    def hackernews_top_stories(hackernews_top_story_ids):
+        """Get items based on story ids from the HackerNews items endpoint"""
+        results = []
+        for item_id in hackernews_top_story_ids:
+            item = requests.get(
+                f"https://hacker-news.firebaseio.com/v0/item/{item_id}.json"
+            ).json()
+            results.append(item)
+
+        df = pd.DataFrame(results)
+
+        # recorded metadata can be customized
+        metadata = {
+            "num_records": len(df),
+            "preview": MetadataValue.md(df[["title", "by", "url"]].to_markdown()),
+        }
+
+        return Output(value=df, metadata=metadata)
+
+
+    defs = Definitions(
+        assets=[hackernews_top_story_ids, hackernews_top_stories],
+        resources={
+            "io_manager": S3PickleIOManager(
+                s3_resource=S3Resource(
+                    region_name="fr-par",
+                    endpoint_url="https://s3.fr-par.scw.cloud",
+                    aws_access_key_id=client.access_key,
+                    aws_secret_access_key=client.secret_key,
+                ),
+                s3_bucket=S3_BUCKET_NAME,
+            ),
+        },
+    )
+    ```
+
+## Setting up the environment and creating the image
+
+1. Retrieve the following elements:
+    - your [Serverless SQL Database connection string](/serverless/sql-databases/how-to/connect-to-a-database/#how-to-set-up-credentials)
+    - the [access key and secret key](/identity-and-access-management/iam/how-to/create-api-keys/) for your API key
+    - the name of the [Object Storage bucket](/storage/object/how-to/create-a-bucket/) you created
+2. Export the environment variables below:
+    ```bash
+    export PG_CONN_STRING=<your_serverless_database_connection_string>
+    export SCW_ACCESS_KEY=<your_scaleway_access_key>
+    export SCW_SECRET_KEY=<your_scaleway_secret_key>
+    export S3_BUCKET_NAME=<your_bucket_name>
+    ```
+
+3. Build and push the image to your container registry namespace:
+    ```bash
+    docker build -t rg.fr-par.scw.cloud/<your-namespace>/dagster-scaleway-example:latest . &&
+    docker push rg.fr-par.scw.cloud/<your-namespace>/dagster-scaleway-example:latest
+    ```
+    <Message type="note">
+    You can find your container registry name and endpoint in the [Scaleway console](https://console.scaleway.com/registry/namespaces)
+    </Message>
+4. Create a `dagster.yaml` file in the same folder to store the configuration and replace the container image with the one you pushed:
+
+  ```yml
+  run_launcher:
+    module: dagster_scaleway
+    class: ScalewayServerlessJobRunLauncher
```

Comment:
note for sls: We should probably move the github repo + pypi project to a Scaleway org. @thomas-tacquet wtyt?